src/lib/components/ScriptureView.svelte

```diff
@@ -20,6 +27,9 @@ TODO:
     /**unique key to use for groupStore modifier*/
     const key = {};
 
+    let firstBookChapter = $globalConfig?.mainFeatures['start-at-reference'];
+    $: $refs = { book: firstBookChapter.split('.')[0], chapter: firstBookChapter.split('.')[1] };
```

Comment:
You're right!!! Thank you. It would make the most sense to set the refs store on startup in +page.svelte, putting it in ScriptureView means it will automatically overwrite the persistently stored reference the user last navigated to. 

perhaps an `onMount()` in +page.svelte or as a default setting of the refs store before its overwritten to the current reference.

Reply:
Here is an idea:

When we read the globalConfigJson, check to see if localStorage.getItem('start-at-reference') is set. If not, then set it to the value from globalConfigJson (not the globalConfig store).

```
if (localStorage.getItem('start-at-reference') == null) {
    localStorage.setItem('start-at-reference', globalConfigJson?.mainFeatures['start-at-reference']);
}
```

In the initializer for the referenceStore, use the value in localStorage.getItem('start-at-reference'). Also set a subscribe that updates localStorage when the reference is changed.

Reply:
Which file does that need to be written in?